R/downloadableReactTable.R

```diff
@@ -148,15 +150,33 @@ downloadableReactTableUI <- function(id,
 #'                             contentHeight = "300px",
 #'                             singleSelect  = TRUE)))),
 #'    server = function(input, output) {
-#'      downloadableReactTable("object_id1")})
+#'        downloadableReactTable(id         = "object_id1",
+#'                               table_data = reactiveVal(mtcars))})
```

Comment:
I do not understand why you would put a default value here.

Reply:
I am not sure what you are referring to Connie, this is the example app and not the function signature